### PR TITLE
UI Enhancements

### PR DESCRIPTION
**Pull request by JeffCX**
I deploy bootstrap and enhance ui of quiz and grading quiz

### CODE REVIEW

**Comment by akshaytambe** [comment]
Please Refer DC -52 for bug fixes! Check with @mkong0216

**Comment by akshaytambe** [comment]
Please Resolve Conflicts

File: devops/templates/graded_quiz.html [hunk 1]
```diff
@@ -4,40 +4,65 @@
...
+
+         {% if navigate_links.previous %}
+            <a href={% url navigate_links.previous %} class=" btn btn-danger" style="margin-bottom:1vh;margin-top:1vh;width:100%;">
                 We suggest you repeat this module
             </a>
             <br/>
         {% endif %}
-        <a href={% url navigate_links.next %}>
+
+         <a href={% url navigate_links.next %} class='btn btn-primary' style="width:100%;">
```
Comment (mkong0216) [inline devops/templates/graded_quiz.html]: Maybe change the blue button to green to mean success?

File: devops/templates/index.html [hunk 2]
```diff
@@ -1,94 +1,154 @@
...
-            <figcaption>
-                The DevOps toolchain
-            </figcaption>
-        </figure>
+<div class="row" style="margin-top:1vh;">
+       <div class="col-sm-4">
+
+               <iframe width="350" height="315" src="https://www.youtube.com/embed/_I94-tJlovg" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+                <img class="d-block w-100" src="https://cdn.pixabay.com/photo/2018/02/12/13/58/devops-3148393_1280.png" alt="Second slide">
```
Comment (mkong0216) [inline devops/templates/index.html]: This is giving me an error image. Please keep that in mind when using images that they might not work.

Are these images you added or the professor originally wanted?

File: devops/templates/index.html [hunk 3]
```diff
@@ -1,94 +1,154 @@
...
-        </ol>
+                 <span class="badge badge-primary badge-pill">12</span>
+          </li>
+        </ul>
+
+</div>
+
+
+<div>
+        <!--
```
Comment (mkong0216) [inline devops/templates/index.html]: If you're commenting out lines, remove them.

File: devops/templates/quiz.html [hunk 4]
```diff
@@ -20,10 +23,13 @@
...
-                            <li>
-                                {{ q.text }}
-                            </li>
-                            <ol class="nested" type='a'>
+
+                             <div class="card" style="margin-top:2vh;font-size:1.5rem;">
+                              <div class="card-body">
+                                <h5 class="card-title" style="font-size:1.5rem;"> {{ q.text }}</h5>
+
+                                <p class="card-text">
```
Comment (mkong0216) [inline devops/templates/quiz.html]: There are extra "card" boxes between the questions and choices and also after the choices. Can you fix those?

**Pull request merged**